metaflow/__init__.py

```diff
@@ -100,6 +100,12 @@ class and related decorators.
 # current runtime singleton
 from .metaflow_current import current
 
+# system monitor runtime singleton (meant for internal use)
+from .metaflow_system_monitor import _system_monitor
```

Comment:
do we need to expose this as a top level import. we can consider doing - `from metaflow.system import monitor`